playbooks/roles/ec2-snapshot/tasks/cron.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: setup cron
+  template:
+    src: cron.j2
+    dest: /etc/cron.d/ec2_snapshot
+    owner: root
+    group: root
+    mode: 0644
+  become_method: sudo
+  become: yes
+  become_user: root
```

Comment:
@sidorovdmitry 

 - It's will be difficult to pars _ec2_snapshot_cron_schedule: 0 4 * * *_ into minute/hours/month/... parameter for ansible cron_module.
 - if we use a template we can add different jobs for different instance types in a future I think